backend-api/infra/monitoring/canaryAlarms.yaml

```
@@ -0,0 +1,656 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  AsyncTokenErrorRateAlarm:
+    Condition: UseLinearCanaryDeployments
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmDescription: "The number of Async Token Lambda errors is greater than or equal to 10% of all function invocations."
```

I saw this below and I quite liked it;

```suggestion
      AlarmDescription: "The number of Async Token Lambda errors is greater than or equal to 10% for the latest function version"
```

Are we ok to roll it out 🥺